docs/usage/deploy-to-different-namespaces.md

```diff
@@ -0,0 +1,14 @@
+# Deploy resources to a different namespace
+
+To grant Argo CD the permissions to manage resources in multiple namespaces, we need to configure the namespace with a label `argocd.argoproj.io/managed-by` and the value being the namespace of the Argo CD instance meant to manage the namespace.
```

Comment:
can we rephrase this as "...value being the namespace of the managing Argo CD instance?" I think its a little less confusing that way